vendor/backward-cpp/CMakeLists.txt

```diff
@@ -77,58 +65,75 @@ if (CMAKE_CXX_COMPILER_ID STREQUAL "Clang" OR CMAKE_COMPILER_IS_GNUCXX)
 endif()
 
 ###############################################################################
-# BACKWARD INTERFACE
-###############################################################################
-
-add_library(backward_interface INTERFACE)
-set_target_properties(backward_interface PROPERTIES EXPORT_NAME Interface)
-target_compile_definitions(backward_interface INTERFACE ${BACKWARD_DEFINITIONS})
-target_include_directories(backward_interface INTERFACE ${BACKWARD_INCLUDE_DIRS})
-if(BACKWARD_HAS_EXTERNAL_LIBRARIES)
-    target_link_libraries(backward_interface INTERFACE ${BACKWARD_LIBRARIES})
-endif()
-add_library(Backward::Interface ALIAS backward_interface)
-
-###############################################################################
-# BACKWARD OBJECT (Includes backward.cpp)
-# (Note that this target is not exported, since CMake currently does not allow
-# exporting an OBJECT library.)
+# BACKWARD OBJECT
 ###############################################################################
 
 add_library(backward_object OBJECT backward.cpp)
-set_target_properties(backward_object PROPERTIES EXPORT_NAME Object)
-target_link_libraries(backward_object PUBLIC Backward::Interface)
-add_library(Backward::Object ALIAS backward_object)
+target_compile_definitions(backward_object PRIVATE ${BACKWARD_DEFINITIONS})
```

Comment:
I think we do need to link against this object for some reason `Backward::Backward` is not enough